msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -141,9 +204,27 @@ Once the app has been installed to a team, you can use [Graph Explorer](https://
 > - Select **Get link to team**.
 > - Copy and save the **groupId** value from the string.
 > - Log into **Graph Explorer**.
-> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The clientAppId field in the response will map to the appId specified in the Teams app manifest.
+> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The `clientAppId` field in the response will map to the `webApplicationInfo.id` specified in the Teams app manifest.
   ![Graph explorer response to GET call.](../../assets/images/graph-permissions.png)
 
+#### See also
+[Get the names and other details of apps installed in the specified team](/graph/api/team-list-installedapps)
```

Comment:
"For information about how to get details about apps installed in a specific team, see <topic title>."